app/components/CapitalProjectsList/CapitalProjectsAccordionPanel.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Flex,
+  Heading,
+} from "@nycplanning/streetscape";
+import { Agency, CapitalProject } from "~/gen";
+import { CapitalProjectsList } from "./CapitalProjectsList";
+
+export interface CapitalProjectsAccordionPanelProps {
+  capitalProjects: Array<CapitalProject>;
+  district: string;
+  agencies: Agency[];
+  children: React.ReactNode;
+}
+
+export const CapitalProjectsAccordionPanel = ({
+  capitalProjects,
+  district,
+  agencies,
+  children,
+}: CapitalProjectsAccordionPanelProps) => {
+  return (
+    <Flex
+      borderRadius={"base"}
+      padding={{ base: 3, lg: 4 }}
+      background={"white"}
+      direction={"column"}
+      width={{ base: "full", lg: "21.25rem" }}
+      maxW={{ base: "21.25rem", lg: "unset" }}
+      boxShadow={"0px 8px 4px 0px rgba(0, 0, 0, 0.08)"}
+      gap={4}
+    >
+      <Accordion defaultIndex={[0]} allowToggle>
+        <AccordionItem border="none">
+          <h2>
```

Comment:
Is there a reason this uses `<h2>` instead of `<Heading>`? I've had issues with that component too but it is meant to replace using the native h1-6 elements using `as=`

Reply:
Nope, I think it was a vestige of initial work, and far as I can tell, we don't even need it. Removing it